xstate-chatbot/react-app/src/diagram.jsx

```
@@ -0,0 +1,109 @@
+
+inspect({
+    iframe: false
+})
+
+function MessageTemplate(props) {
+    let message = props.item.text;
+    message = message.replaceAll('\n', '<br/>');
+    let htmlToinsert = { __html: message };
+    return (
+        <div className="k-bubble">
+            <div dangerouslySetInnerHTML={htmlToinsert} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)